### PR TITLE
jssrc2cpg: fixed argument indizes in push calls within desugared array literals

### PR DESCRIPTION
**Pull request by max-leuthaeuser**
Fixes: https://github.com/joernio/joern/issues/2049

### CODE REVIEW

File: joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala [hunk 1]
```diff
@@ -1527,19 +1527,15 @@ class SimpleAstCreationPassTest extends AbstractPassTest {
 
     val pushCallReceiverBase = pushCallReceiver.argument(1).asInstanceOf[Identifier]
     pushCallReceiverBase.name shouldBe "_tmp_0"
-    pushCallReceiverBase.argumentIndex shouldBe 1
```
Comment (maltek) [inline joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala:1530]: has this changed? afaict this is the `_tmp_0` within the `_tmp_0.push` field access, where arg-index 1 seems correct

File: joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala [hunk 2]
```diff
@@ -1527,19 +1527,15 @@ class SimpleAstCreationPassTest extends AbstractPassTest {
...
 
     val pushCallReceiverMember = pushCallReceiver.argument(2).asInstanceOf[FieldIdentifier]
     pushCallReceiverMember.canonicalName shouldBe "push"
-    pushCallReceiverMember.argumentIndex shouldBe 2
```
Comment (maltek) [inline joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala:1534]: same question

File: joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala (hunk 1 above)
Comment (max-leuthaeuser) [reply joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala:1530]: Its already in `pushCallReceiver.argument(1).asInstanceOf[Identifier]`.
I just removed the double testing there.

File: joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala (hunk 2 above)
Comment (max-leuthaeuser) [reply joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala:1534]: same answere :)

File: joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala (hunk 1 above)
Comment (maltek) [reply joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala:1530]: duh, of course

**Pull request merged**